package site.dogether.challengegroup.exception;

public class MemberAlreadyInChallengeGroupException extends RuntimeException {
    public MemberAlreadyInChallengeGroupException(String message) {
        super(message);
    }
}
